### ISSUE TITLE
Enabling References::ProhibitDoubleSigils perlcritic policy

### ISSUE THREAD

**Issue by Julien-Elie** [rra/rra-c-util#19]
https://github.com/rra/rra-c-util/blob/88eed5bf1758ffd9cf53627e600fc09557ebdeb7/tests/data/perlcriticrc#L92-L94

The issue was fixed upstream, so it may be enabled now.

**Comment by rra** [comment]
Yeah, I've finally given in and am starting to standardize everything on one space after periods. I decided that I wanted to prove to myself that I could change and retrain my fingers to use the style that all the kids use these days. :)

Although I guess that's kind of awkward for git blame and the like, so maybe I should reconsider where I make that change! That just occurred to me.
